.buildkite/build.sh

```diff
@@ -46,7 +46,7 @@ echo "--- Uploading"
 
 
 # Clear dist so that the dmg is in the same dir as the rest of the packages
-rm -r dist/* && mv package/osx/kolibri*.dmg dist/
+rm -r dist/* && mv package/osx/kolibri*.dmg dist/kolibri-$(more src/kolibri/VERSION)-$(git rev-parse --short HEAD).dmg
```

Comment:
would this capture either tagged releases or commit hashes?

```suggestion
rm -r dist/* && mv package/osx/kolibri*.dmg dist/kolibri-$(more src/kolibri/VERSION)-$(git describe --exact-match --tags || git rev-parse --short HEAD).dmg
```

Reply:
Tagged releases or commit hashes in the macOS repo? No.

I could add that once we do start tagging things, though.

Or I can do it now. Might take a bit, though. 

> also recommend adding macos- prefix to mac version info

👌